apps/condo/domains/notification/schema/SendMessageService.js

```diff
@@ -76,6 +83,19 @@ async function checkSendMessageMeta (type, meta, context) {
     }
 }
 
+/**
+ * Gets condo tasks queue on which to start message delivery
+ * @param {string} type Message type
+ * @returns {'medium' | 'high' | 'low'} condo task queue
+ */
+function getMessageQueue ({ type }) {
+    // TODO(VKislov): separate queue for VoIP here?
+    const defaultQueue = MESSAGE_DELIVERY_PRIORITY_TO_TASK_QUEUE_MAP[MESSAGE_DELIVERY_DEFAULT_PRIORITY]
+    const messagePriority = get(MESSAGE_DELIVERY_OPTIONS, [type, 'priority'], MESSAGE_DELIVERY_DEFAULT_PRIORITY)
+
+    return get(MESSAGE_DELIVERY_PRIORITY_TO_TASK_QUEUE_MAP, messagePriority, defaultQueue)
```

Comment:
There is already a default queue in the `tasks` package. Why don't you to reuse it?

Reply:
There's small difference in logic:
1. Your suggestion is - use default app queue as default.
2. Mine is - use **same queue** as **default priority uses**

For now they're the same, since they're both pointing to `medium/default`. But IMHO the second option is more accurate in terms of possible future changes

Reply:
Ok, sounds reasonable